### ISSUE TITLE
babel plugin tries to transform Select element from other than react-lingui library

### ISSUE THREAD

**Issue by vitexikora** [lingui/js-lingui#52]
Extract command crashed on one of my components (helper one, without any translations), had to remove it from parsing via ignore pattern. The reason it is crashing is completely strange to me.
The entire file (`src\components\helpers\SearchableSelect.js`):
```js
import React, { PureComponent } from 'react'
import cls from '../../../styles/layout.less'
import InputLabel from 'material-ui/Input/InputLabel'
import FormControl from 'material-ui/Form/FormControl'
import FormHelperText from 'material-ui/Form/FormHelperText'
import Select from 'react-select'
import 'react-select/dist/react-select.css'

export default class SearchableSelect extends PureComponent {
  render() {
    const {name, value, options, label, helperText, changeHandler} = this.props
    let labelProps = {}
    if (value) {
      labelProps.shrink = true
    }
    return (
      <FormControl className={cls.formControl + ' ' + cls.selectField}>
        <InputLabel htmlFor={name} className={cls.inputLabel} {...labelProps}>{label}</InputLabel>
        <Select 
          className={cls.select}
          name={name}
          value={value}
          options={options}
          placeholder=''
          noResultsText={false}
          onChange={changeHandler}
        />
        {helperText && <FormHelperText className={cls.helperText}>{helperText}</FormHelperText>}
      </FormControl>
    )
  }
}
```

The error (@ `lingui-cli\node_modules\babel-core\lib\transformation\file\index.js:590`):
```
SyntaxError: ... /src/components/helpers/SearchableSelect.js: Missing fallback argument 'other'.
  17 |       <FormControl className={cls.formControl + ' ' + cls.selectField}>
  18 |         <InputLabel htmlFor={name} className={cls.inputLabel} {...labelProps}>{label}</InputLabel>
> 19 |         <Select
     |         ^
  20 |           className={cls.select}
  21 |           name={name}
  22 |           value={value}
    at File.buildCodeFrameError (lingui-cli\node_modules\babel-core\lib\transformation\file\index.js:427:15)
    at PluginPass.buildCodeFrameError (lingui-cli\node_modules\babel-core\lib\transformation\plugin-pass.js:64:53)
    at processElement (lingui-cli\node_modules\babel-plugin-lingui-transform-react\dist\index.js:186:20)
    at PluginPass.JSXElement (lingui-cli\node_modules\babel-plugin-lingui-transform-react\dist\index.js:396:21)
    at newFn (lingui-cli\node_modules\babel-traverse\lib\visitors.js:276:21)
    at NodePath._call (lingui-cli\node_modules\babel-traverse\lib\path\context.js:76:18)
    at NodePath.call (lingui-cli\node_modules\babel-traverse\lib\path\context.js:48:17)
    at NodePath.visit (lingui-cli\node_modules\babel-traverse\lib\path\context.js:105:12)
    at TraversalContext.visitQueue (lingui-cli\node_modules\babel-traverse\lib\context.js:150:16)
    at TraversalContext.visitMultiple (lingui-cli\node_modules\babel-traverse\lib\context.js:103:17)
    at TraversalContext.visit (lingui-cli\node_modules\babel-traverse\lib\context.js:190:19)
    at Function.traverse.node (lingui-cli\node_modules\babel-traverse\lib\index.js:114:17)
```

**Comment by tricoder42** [comment]
It's because component of the same name as Select is used in react-trans and parsed in babel plugin. The plugin doesn't check from where the component is included.

Good catch! I'll fix it once I get back from vacation

8. 8. 2017 v 13:10, vitexikora <notifications@github.com>:

**Comment by tricoder42** [comment]
Fixed in babel-preset-lingui-react@**1.0.2** and babel-plugin-lingui-transform-react@**1.0.2**
